src/ui/server/controllers/insurance/your-buyer/company-or-organisation/index.test.ts

```diff
@@ -192,6 +192,20 @@ describe('controllers/insurance/your-buyer/company-or-organisation', () => {
 
         expect(mapAndSave.yourBuyer).toHaveBeenCalledWith(req.body, mockApplication);
       });
+
+      describe("when the url's last substring is `change`", () => {
+        it(`should redirect to ${CHECK_YOUR_ANSWERS}`, async () => {
+          const { exporterIsConnectedWithBuyer, exporterHasTradedWithBuyer, ...companyOrOrganisationMock } = mockBuyer;
+          req.body = companyOrOrganisationMock;
```

Comment:
could we have `mock` at the beginning to be aligned 

`mockCompanyOrOrganisation`